index.html

```diff
@@ -56,52 +56,58 @@ <h1 class="title">
-            <div class="text-card">
-                <h2 class="title cards">Tonic</h2>
-                <img src="images/fame2.png" alt="frame" class="frame">
+            <div class="text-card two">
+                <h2 class="title cards tonic">Tonic</h2>
+                <img src="images/fame2.png" class="frame frame1">
                 <p class="text">
                     A daily selection of privately personalized reads; no accounts or sign-ups required.
                 </p>
-                <ul>
+                <ul class="code-links">
                     <li class="tag">
-                        <a href="#">html</a>
+                        <a href="" class="a-tag">html</a>
                     </li>
                     <li class="tag">
-                        <a href="#">css</a>
+                        <a href="" class="a-tag">css</a>
                     </li>
                     <li class="tag">
-                        <a href="#">javaScript</a>
+                        <a href="" class="a-tag">javaScript</a>
                     </li>
                 </ul>
                 <a href="#" class="button-project" onclick="openPopup(0)">See project</a>
             </div>
         </div>
 
+
         <div class="card card-two">
             <img src="images/portfolio2.png" alt="a_screenshot" class="multi screen-pic">
             <div class="text-card two">
                 <h2 class="title cards multi">Multi-Post Stories</h2>
                 <img src="images/fame2.png" class="frame frame2">
-                <p class="text">
-                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                <p class="text text-multi">
+                    Experimental content creation feature that allows users to add to an existing story over the course of a day without spamming their friends.
                 </p>
-                <ul class="code-links">
+                <ul class="code-links-multi">
                     <li class="tag">
-                        <a href="">html</a>
+                        <a href="" class="a-tag">html</a>
                     </li>
                     <li class="tag">
-                        <a href="">css</a>
+                    <a href="" class="a-tag">Ruby</a>
+                </li>
+                    <li class="tag">
+                        <a href="" class="a-tag">css</a>
                     </li>
+                </li>
                     <li class="tag">
-                        <a href="">javaScript</a>
+                        <a href="" class="a-tag">javaScript</a>
                     </li>
                 </ul>
-                <a href="#" class="button-project" onclick="openPopup(1)">See project</a>
+                <a href="#" class="button-project b-multi" onclick="openPopup(1)">See project</a>
             </div>
         </div>
     </section> 
```

Comment:
- [ ] kindly consider removing this section from your html file  and create it  dynamically with the information from the projects array you used in your pop-up.js(all the HTML of the section is created when the page loads) to meet the below circled requirement
<img width="1040" alt="Screen Shot 2023-07-26 at 17 16 24" src="https://github.com/migcm06/miguel-portfolio/assets/60146030/5ae85ea2-aaba-41c1-82de-c4de065fa4ec">